.github/workflows/static-checks-npm.yml

```diff
@@ -45,3 +45,11 @@ jobs:
         run: npm ci
       - name: ${{ matrix.command }}
         run: npm run ${{ matrix.command }}
+      - name: TruffleHog PR scan
+        uses: trufflesecurity/trufflehog@v3.90.12
+        if: ${{ always() }}
+        with:
+          base: ""
+          head: ${{ github.ref_name }}
+          extra_args: --results=verified,unknown
+        continue-on-error: true
```

Comment:
This should be a separate job in that static-checks workflow, rather than just bolted onto the matrix job.